core/src/log/mod.rs

```diff
@@ -16,6 +17,8 @@ mod logger;
     }
 }
 
+pub static LOGGING_INITIALIZED: AtomicBool = AtomicBool::new(false);
```

Comment:
on a second thought, it might be better to keep it private and have only a public getter in the module